imports/plugins/core/catalog/server/no-meteor/utils/publishProductToCatalog.js

```diff
@@ -19,22 +21,31 @@ export default async function publishProductToCatalog(product, collections) {
   // Convert Product schema object to Catalog schema object
   const catalogProduct = await createCatalogProduct(hashedProduct, collections);
 
+  const modifier = {
+    $set: {
+      product: catalogProduct,
+      shopId: catalogProduct.shopId,
+      updatedAt: new Date()
+    },
+    $setOnInsert: {
+      _id: Random.id(),
```

Comment:
@kieckhafer @aldeed this line appears to be causing problems during startup. Details https://github.com/reactioncommerce/reaction/issues/4434